tests/e2e/lib/virt_storage_helpers.go

```diff
@@ -323,3 +324,27 @@ func (v *VirtOperator) CloneDisk(sourceNamespace, sourceName, cloneNamespace, cl
 
 	return nil
 }
+
+// Create a DataVolume from a sample YAML template. The namespace argument
+// should match a subdirectory under sample-applications/virtual-machines, and
+// the name argument should match a .yaml file in that directory, for example:
+//
+//	sample-applications/virtual-machines/cirros-test/cirros-test-disk.yaml
+//
+// This file should create a DataVolume with the following annotations set:
```

Comment:
since this specified in yaml, I would remove from doc. Afraid we update yaml, but not the doc in the future

Reply:
Okay, I changed the explicit reference to cirros-test.